packages/data-table-server/src/dataTableService/services/AnalyticsDataTableService.ts

```diff
@@ -65,6 +66,13 @@ export class AnalyticsDataTableService extends DataTableService<
     const startDateString = startDate.toISOString();
     const endDateString = endDate.toISOString();
 
+    // Ensure that if fetching for project, we map it to the underlying countries
+    const entityCodesForFetch = await mapProjectEntitiesToCountries(
+      this.ctx.apiClient,
+      hierarchy,
+      organisationUnitCodes,
+    );
+
```

Comment:
You mean build params in the `report-server`? I see your point, and I did consider keeping it around, but there was a couple reasons I didn't want to.
1. Hopefully, we will remove the report-server all together once data-tables is finished, so the more logic we take out of there the better
2. It's actually quite possible we would want to fetch for project entities for various data-tables (eg. the entities or entity relations data-tables). I think leaving it up to each service to decide how they handle entities would be best.

Honestly, I'd prefer if this translations was done in data-broker or something rather than at this level. It doesn't seem appropriate that we do this first, but I didn't wanna do that refactor right now...

Reply:
Yeah good point. No worries, it is out of scope.